apps/storybook/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "storybook",
+  "version": "0.0.0",
+  "private": true,
+  "dependencies": {},
+  "devDependencies": {
+    "@itwin/itwinui-icons-react": "^1.9.0",
+    "@itwin/itwinui-react": "*",
+    "@storybook/addon-a11y": "~6.4.22",
+    "@storybook/addon-actions": "~6.4.22",
+    "@storybook/addon-essentials": "~6.4.22",
+    "@storybook/addon-storysource": "~6.4.22",
+    "@storybook/builder-vite": "^0.1.33",
+    "@storybook/node-logger": "~6.4.22",
+    "@storybook/react": "~6.4.22",
+    "@types/node": "^16.11.31",
+    "@types/react": "^17.0.3",
+    "@types/react-dom": "^17.0.3",
+    "configs": "*",
+    "cypress": "^9.6.0",
+    "cypress-image-diff-js": "^1.18.1",
+    "eslint": "^8.14.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-storybook": "^0.5.7",
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0",
+    "serve": "^13.0.2",
+    "storybook-dark-mode": "^1.1.0",
+    "typescript": "^4.6.3",
+    "vite": "^2.8.0"
+  },
+  "scripts": {
+    "dev:vite": "vite",
```

Comment:
Why `:vite` postfix?

Reply:
because we dont really care about vite commands. I could remove them completely but left them just in case

Reply:
Do we even need them and do they work?

Reply:
They work but the vite app is just hello world. Removed